.github/workflows/wpt-mapper.yml

```diff
@@ -11,6 +11,9 @@ on:
 jobs:
   wpt:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        head: [wpt, wpt-headful]
```

Comment:
Consider [adding configurations](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations):
```
    strategy:
      matrix:
        include:
          - head: "headful"
            command_prefix: "xvfb-run --auto-servernum"
            wpt_argument: "--webdriver-arg="--headless=false"
          - head: "headless"
            command_prefix: ""
            wpt_argument: "--webdriver-arg="--headless=true"
```